Patterns/English/English-DateTime.yaml

```diff
@@ -2,7 +2,9 @@
 TillRegex: !simpleRegex
   def: (?<till>to|till|until|thru|through|--|-|—|——)
 AndRegex: !simpleRegex
-  def: (?<and>and|--|-|—|——)
+  def: (?<and>and|through|--|-|—|——)

```

Comment:
Shouldn't this also cover the case "from D1 to D2"?
If AndRegex is only used in range cases, then it should be rename to RangeConnectorRegex or something similar.